src/extension.ts

```diff
@@ -68,6 +77,17 @@ export async function activate(context: vscode.ExtensionContext) {
     const treeView = vscode.window.createTreeView("apiCenterTreeView", { treeDataProvider });
     context.subscriptions.push(treeView);
 
+    AzureDataSessionProviderHelper.activateAzureSessionProvider(context);
```

Comment:
Better to wrap those initialazation logic for data plane into a function

Reply:
wrapper both for data plane and control plane